src/Infrastructure/OpcUa/ComplexTypeSystemWrapper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using OMP.Connector.Domain;
+using Opc.Ua.Client;
+using Opc.Ua.Client.ComplexTypes;
+
+namespace OMP.Connector.Infrastructure.OpcUa
+{
+    public class ComplexTypeSystemWrapper: ComplexTypeSystem, IComplexTypeSystem
+    {
+        public ComplexTypeSystemWrapper(Session session)
```

Comment:
Hopefully after the 'ISession' is release in the Opc UA Foundation nuget we can use the ISession interface here